spec/models/defence_request_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe DefenceRequest, type: :model do
+  subject { DefenceRequest.new }
```

Comment:
If I remove the settng of subject then `:validations` becomes the default subject of its block